contracts/test/NonStandardFaucetFeeToken.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+/**
+ * @title Non-standard ERC20 token
+ * @dev Implementation of the basic standard token.
+ *  See https://github.com/ethereum/EIPs/issues/20
+ * @dev With USDT fee token mechanism
+ * @dev Note: `transfer` and `transferFrom` do not return a boolean
+ */
+contract NonStandardFeeToken {
```

Comment:
Can you make this extend the `IERC20NonStandard` just to make sure the interfaces match?

Reply:
np 👍